packages/botonic-pulumi/src/index.ts

```diff
@@ -2,10 +2,27 @@ import { Config } from '@pulumi/pulumi'
 import { join } from 'path'
 import { cwd } from 'process'
 
-export const getNamePrefix = (): string => {
+export function getNamePrefix(): string {
   const config = new Config()
-  return `botonic-${config.get('projectName')}-${config.get('stackName')}`
+  const prefix = generatePrefix(
+    config.get('projectName') as string,
+    config.get('stackName') as string
+  )
+  return prefix
```

Comment:
Unless you're leaving it like this for debugging reasons, you can return the call directly, no need of the variable.